src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -812,13 +812,13 @@ private static String processPageSourceForGodMode(String content) {
                 .replaceAll(Const.ParamsNames.REGKEY + "%3D([a-zA-Z0-9]){1,}\\%", Const.ParamsNames.REGKEY + "%3D\\${regkey\\.enc}\\%")
                 
                 // maintain order for the two below
-                // questionid
-                .replaceAll("value=\"([a-zA-Z0-9-_]){62,}\"","value=\"\\${question\\.id}\"")
                 // regkey in unreg student page
-                .replaceAll("value=\"([a-zA-Z0-9-_]){50,}\"","value=\"\\${regkey\\.enc}\"")
+                .replaceAll("(type=\"hidden\" ?|name=\""+Const.ParamsNames.REGKEY+"\" ?|value=\"([a-zA-Z0-9-_]){1,}\" ?){3}","name=\""+Const.ParamsNames.REGKEY+"\" type=\"hidden\" value=\"\\${regkey\\.enc}\"")
```

Comment:
The params are randomized, is it? I thought God mode acts after it's converted to standard HTML, which means the params should've been sorted already.
Also, I think the regex for regkey shouldn't be softened. 50 as before works good. Unless there's a reason?